src/kudu/consensus/raft_consensus.cc

```diff
@@ -2353,15 +2381,16 @@ Status RaftConsensus::CheckBulkConfigChangeAndGetNewConfigUnlocked(
             // requirement, because it gives flexibility to automation to replace nodes without
             // impacting the write availability.
             if (FLAGS_enable_flexi_raft) {
-              const auto& vd_map = committed_config.voter_distribution();
+              std::map<std::string, int> vd_map;
```

Comment:
potentially add a quorum id based test for this or modify existing test to work
in quorum_id mode as well